### PR TITLE
feat(*): add dependency resolution

### PR DESCRIPTION
**Pull request by technosophos**
Addresses #9, #75. See #58.

### CODE REVIEW

**Comment by adamreese** [comment]
`dependency` package needs to be added to `make test`

File: helm/action/info_test.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ func init() {
 }
 
 func TestInfo(t *testing.T) {
+	t.Skip()
```
Comment (mboersma) [inline helm/action/info_test.go]: There should be a comment here noting why this test is skipped.

File: helm/dependency/dependency.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/deis/helm/helm/log"
+	"github.com/deis/helm/helm/model"
+)
+
+// ResolveDependencies takes a chart and a location and checks whether the chart's dependencies are satisfied.
```
Comment (mboersma) [inline helm/dependency/dependency.go]: This comment should start with `Resolve takes a chart`...

**Comment by technosophos** [comment]
Fixed the issues @mboersma and @adamreese noted.

File: helm/dependency/dependency.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+// The `installdir` is the location where installed charts are located. Typically
+// this is in $HELM_HOME/workspace/charts.
+//
+// This returns a list of unsatisfied dependencies (NOT an error condition).
+//
+// It returns an error only if it cannot perform the task of resolving dependencies.
+// Failed dependencies to not constitute an error.
+func Resolve(cf *model.Chartfile, installdir string) ([]*model.Dependency, error) {
+	if len(cf.Dependencies) == 0 {
+		log.Debug("No dependencies to check. :achievement-unlocked:")
```
Comment (bacongobbler) [inline helm/dependency/dependency.go]: +1

**Pull request merged**